hera_cal/chunker.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright 2021 the HERA Project
+# Licensed under the MIT License
+
+
+from . import io
+import argparse
+import numpy as np
+import sys
+import warnings
+from pyuvdata import utils as uvutils
+
+
+def chunk_files(filenames, inputfile, outputfile, chunk_size, type="data",
+                polarizations=None, spw_range=None, throw_away_flagged_ants=False,
+                clobber=False, ant_flag_yaml=None):
+    """Chunk a list of data or cal files together into a single file.
+
+    Parameters
+    ----------
+    filenames: list of strings
+        list of filenames to chunk. Should be homogenous in blt.
+    outpufile: str
```

Comment:
Just realized that `inputfile` doesn't show up in this list either.

Reply:
Thanks for catching this! I've added it.